editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1258,6 +1258,11 @@ export const MetadataUtils = {
     if (isImported) {
       return false
     }
+    const isImportedFromUtopiaApi =
+      element != null ? isUtopiaAPIComponentFromMetadata(element) : false
```

Comment:
Maybe it is just me, but for me `element == null || isUtopiaAPIComponentFromMetadata(element)` is way easier to process than this ternary.
Maybe because there are just too many negations in the original?